app/src/main/java/org/p2p/wallet/history/ui/token/adapter/HistoryAdapter.kt

```diff
@@ -43,25 +46,35 @@ class HistoryAdapter(
             HISTORY_DATE_VIEW_TYPE -> DateViewHolder(parent)
             PROGRESS_VIEW_TYPE -> ProgressViewHolder(parent)
             TRANSACTION_MOONPAY_VIEW_TYPE -> HistorySellTransactionViewHolder(parent, onHistoryItemClicked)
+            TRANSACTION_USER_SEND_LINKS_VIEW_TYPE -> HistoryUserSendLinksViewHolder(parent, onHistoryItemClicked)
             else -> ErrorViewHolder(parent)
         }
     }
 
     override fun onBindViewHolder(holder: HistoryTransactionViewHolder, position: Int) {
         when (holder) {
-            is TransactionViewHolder -> holder.onBind(getItem(position) as TransactionItem)
-            is TransactionSwapViewHolder -> holder.onBind(getItem(position) as TransactionItem)
-            is DateViewHolder -> holder.onBind(getItem(position) as DateItem)
-            is ErrorViewHolder -> holder.onBind(onRetryClicked)
-            is HistorySellTransactionViewHolder -> holder.onBind(getItem(position) as MoonpayTransactionItem)
-            is ProgressViewHolder -> Unit
+            is TransactionViewHolder ->
+                holder.onBind(getItem(position) as TransactionItem)
+            is TransactionSwapViewHolder ->
+                holder.onBind(getItem(position) as TransactionItem)
+            is DateViewHolder ->
+                holder.onBind(getItem(position) as DateItem)
+            is ErrorViewHolder ->
+                holder.onBind(onRetryClicked)
+            is HistorySellTransactionViewHolder ->
+                holder.onBind(getItem(position) as MoonpayTransactionItem)
+            is ProgressViewHolder ->
+                Unit
+            is HistoryUserSendLinksViewHolder ->
+                holder.onBind(getItem(position) as UserSendLinksItem)
```

Comment:
Are you sure these line don't fit the single line?

Reply:
they fit, but what looks better? 
<img width="653" alt="Screen Shot 2023-03-31 at 18 57 18" src="https://user-images.githubusercontent.com/100791035/229102425-5c649e44-449e-47f6-869f-16e6139d7aeb.png">
or
<img width="797" alt="Screen Shot 2023-03-31 at 18 57 29" src="https://user-images.githubusercontent.com/100791035/229102439-128e70c6-e51c-4cc3-8412-3a00d9e862cc.png">